packages/contracts/src/contracts/current/protocol/Exchange/MixinWrapperFunctions.sol

```diff
@@ -141,54 +126,38 @@ contract MixinWrapperFunctions is
     /// @param orderAddresses Array of address arrays containing individual order addresses.
     /// @param orderValues Array of uint256 arrays containing individual order values.
     /// @param takerTokenFillAmounts Array of desired amounts of takerToken to fill in orders.
-    /// @param v Array ECDSA signature v parameters.
-    /// @param r Array of ECDSA signature r parameters.
-    /// @param s Array of ECDSA signature s parameters.
     function batchFillOrKillOrders(
         address[5][] orderAddresses,
-        uint256[6][] orderValues,
-        uint256[] takerTokenFillAmounts,
-        uint8[] v,
-        bytes32[] r,
-        bytes32[] s)
-        external
+        uint[6][] orderValues,
+        uint[] takerTokenFillAmounts,
+        bytes[] signatures)
+        public /// Compiler crash when set to external
     {
         for (uint256 i = 0; i < orderAddresses.length; i++) {
             fillOrKillOrder(
                 orderAddresses[i],
                 orderValues[i],
                 takerTokenFillAmounts[i],
-                v[i],
-                r[i],
-                s[i]
+                signatures[i]
             );
         }
     }
 
-    /// @dev Synchronously executes multiple calls of fillOrderNoThrow in a single transaction.
-    /// @param orderAddresses Array of address arrays containing individual order addresses.
-    /// @param orderValues Array of uint256 arrays containing individual order values.
-    /// @param takerTokenFillAmounts Array of desired amounts of takerToken to fill in orders.
-    /// @param v Array ECDSA signature v parameters.
-    /// @param r Array of ECDSA signature r parameters.
-    /// @param s Array of ECDSA signature s parameters.
-    function batchFillOrdersNoThrow(
+
+    function fillOrdersUpTo(
```

Comment:
Remove this function, we renamed to `marketFillOrders`.

Reply:
Ignore my previous comment, looks like you accidentally renamed `batchFillOrdersNoThrow` to `fillOrdersUpTo`.